Alloy/commands/compile/ast/controller.js

```diff
@@ -6,16 +6,20 @@ var U = require('../../../utils'),
 
 var isBaseControllerExportExpression = types.buildMatchMemberExpression('exports.baseController');
 
-const GENCODE_OPTIONS = {
+var GENCODE_OPTIONS = {
```

Comment:
We should probably use the modern equivalent:

```suggestion
let GENCODE_OPTIONS = {
```